src/components/PostComment/PostComment.jsx

```diff
@@ -0,0 +1,91 @@
+import axios from 'axios';
+import { useState, useContext } from 'react';
+import './PostComment.css';
+import { DefaultUserContext } from '../../Context/DefaultUserContext';
+
+const PostComment = ({
+	review_id,
+	allComments,
+	setAllComments,
+}) => {
+	const [error, setError] = useState();
+	const [isSubmitted, setIsSubmitted] = useState(false);
+	const [body, setBody] = useState('');
+	const { user } = useContext(DefaultUserContext);
+
+	const SubmitComment = (event) => {
```

Comment:
use proper camelCase for non-React-Component functions

```diff
- SubmitComment
+ submitComment
```

Reply:
this still needs changing to  submitComment please